configuration/openmrs/apps/reports/sql/testCount.sql

```diff
@@ -0,0 +1,14 @@
+SELECT DISTINCT
```

Comment:
Does this file have any use?

Reply:
Not for easyDoc (for now). It's a default bahmni file and is used for [a lab report ](https://github.com/Bahmni/default-config/blob/master/openmrs/apps/reports/reports.json#L167) for counting lab tests. 
[I've removed the file.](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/14/commits/6a6330730a27226c8a8ebcc0289d129b2be6e2f9)